vpc-cockroachdb-mzr/variables.tf

```diff
@@ -70,15 +70,28 @@ variable "vpc_database_image_name" {
 
 variable "cockroachdb_binary_url" {
   description = "The url for the cockroacdb download."
-  default = "https://binaries.cockroachdb.com"
+  default     = "https://binaries.cockroachdb.com"
 }
 
 variable "cockroachdb_binary_archive" {
   description = "The archive filename for the cockroacdb download."
-  default = "cockroach-v20.2.4.linux-amd64.tgz"
+  default     = "cockroach-v20.2.4.linux-amd64.tgz"
 }
 
 variable "cockroachdb_binary_directory" {
   description = "The directory for the cockroacdb archive when extracted."
-  default = "cockroach-v20.2.4.linux-amd64"
-}
\ No newline at end of file
+  default     = "cockroach-v20.2.4.linux-amd64"
+}
+
+variable "create_secrets_manager_instance" {
+  description = "Indicates whether or not to create a Secrets Manager instance, values can be true or false, default to false."
+  default     = false
+}
+
+variable "secrets_manager_instance_name" {
+  description = "The name of an existing Secrets Manager instance, used when create_secrets_manager_instance is set to false."
+}
+
+variable "secrets_manager_group_name" {
```

Comment:
same for the group name. If you create a new instance, the name could have a good default.

Reply:
added